milestones/tests/test_services.py

```diff
@@ -0,0 +1,39 @@
+"""
+Test for the xBlock service
+"""
+
+import unittest
+import types
+
+from milestones.services import (
+    MilestonesService
+)
+from milestones import api as milestones_api
+
+
+class TestMilestonesService(unittest.TestCase):  # pylint: disable=too-many-public-methods
```

Comment:
Pylint only allows for about 20 (?) public methods in a class. It considers each `assert` to be a public method. Since `test_basic` iterates through each function in `milestones_api` and asserts whether or not it should be in `MilestonesService`, it's more than the allowed public methods limit.